### DISCUSSION TITLE
Kernel Problem

### DISCUSSION THREAD

**Discussion by Adriannaluger** [earthlab-education/Earth-Analytics-2023-01-Intro#502] [Q&A06 Vector Data]
When I open my personal site codespace from my own github repository I get this message:
"The version of Python associated with the selected kernel is no longer supported. Please consider selecting a different kernel."
(I am in the same kernel we have been using all year) Everything still appears to be running fine, should I ignore?

**Comment by eculler** [comment]
Hi @Adriannaluger, hm, I guess that might have to do with the recent release of Python 12? In any case I haven't encountered this issue myself, so I'll need some more info.

Does it actually not run your code, or are you merely getting a warning? If it's just a warning you could ignore it for now and we can get you updated in the future.

If not, you may need to switch over to the newer environment. To do so, you will need to edit your .devcontainer.json file in your personal portfolio repository and then rebuild the container. The edit you need to make is to replace earth-analytics-python-env with earth-analytics-dev-env. You could alternatively copy over the .devcontainer.json from a more recent assignment.

**Reply by Adriannaluger** [reply]
Yes, it is just a warning and my codes are still running as usual so I will ignore it, just thought I would let you know. Thanks!
